test/http-e2e/testdata/http-source.yaml

```diff
@@ -3,6 +3,8 @@ kind: Pipeline
 metadata:
   name: http-source
 spec:
+  watermark:
```

Comment:
do we have one with watermark on?

Reply:
In all the other tests watermark is enabled